README.md

```diff
@@ -130,13 +133,11 @@ To run the project, execute the following command:
 
 To run tests, run the following command:
 
-- This project doesn't have tests
+- This project doesn't have any tests yet
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** I think it would be better if you add a command of how to run your tests as your project has tests now. This will improve your documentation to be more professional and descriptive